src/Control/Effect/Exception.hs

```diff
@@ -0,0 +1,254 @@
+{-# LANGUAGE DeriveFunctor, ExistentialQuantification, RankNTypes, StandaloneDeriving #-}
+-- | Operations from "Control.Exception" lifted into effectful contexts using 'Control.Effect.Lift.Lift'.
+--
+-- @since 1.0.0.0
+module Control.Effect.Exception
+( -- * Lifted "Control.Exception" operations
+  throwIO
+, ioError
+, throwTo
+, catch
+, catches
+, Handler(..)
+, catchJust
+, handle
+, handleJust
+, try
+, tryJust
+, evaluate
+, mask
+, mask_
+, uninterruptibleMask
+, uninterruptibleMask_
+, getMaskingState
+, interruptible
+, allowInterrupt
+, bracket
+, bracket_
+, bracketOnError
+, finally
+, onException
```

Comment:
I’ve tried to order this, and the definitions below, to match the `Control.Exception` haddocks.